test/Rhetos.LightDMS.IntegrationTest/DownloadS3StorageTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using Rhetos.LightDMS.IntegrationTest.Utilities;
+using Rhetos.LightDMS.TestApp;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Rhetos.LightDMS.IntegrationTest
+{
+    public class RhetosEntiy
+    {
+        public string ID { get; set; }
+    }
+
+    [Collection("S3 Storage simulator - local files")]

```

Comment:
Collection name is a bit misleading, it is remote file, not local files

Reply:
Removed it.